recipe/patches/0027-Set-ssltag-to-3.patch

```diff
@@ -0,0 +1,39 @@
+From eb3e4e0dbd5a925825cd2fb989033d8b1860cfc0 Mon Sep 17 00:00:00 2001
+From: Chris Burr <chrisburr@users.noreply.github.com>
+Date: Tue, 28 Sep 2021 12:40:44 +0200
+Subject: [PATCH 29/29] Set ssltag to 3
```

Comment:
What's the error you get when this happens?

Reply:
On Windows it tries to link to the wrong shared library name (see the CI in the Python 3.9 OpenSSL PR)